Engine Hacks/SkillSystem/Other Skills/shadowgift.s

```diff
@@ -2,23 +2,26 @@
 .equ LuminaID, ShadowgiftID+4
 .equ ShadowgiftStaffOption, LuminaID+4
 .equ LuminaStaffOption, ShadowgiftStaffOption+4
+.equ ItemTable,LuminaStaffOption+4
 .thumb
 
-push	{r4-r6}
+push	{r14}
 
 @r4 = character pointer
-mov	r5,r1	@item type
-mov	r6,r2	@rank
+@r5 = item halfword
+@r6 = rank
```

Comment:
See above (if review comment order is consistent?) comment on calling conventions.